glob.go

```diff
@@ -48,10 +53,14 @@ func QuoteMeta(pattern string) string {
 	return glob.QuoteMeta(pattern)
 }
 
+func CleanPath(path string) string {
```

Comment:
maybe could be private?

Reply:
Yes, there is not much of value for the user in this function, as anyone can just chain these two functions together easily anyway.